spring-data-azure-documentdb/src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplate.java

```diff
@@ -285,6 +286,10 @@ private String getDatabaseLink(String databaseName) {
     }
 
     private String getCollectionLink(String databaseName, String collectionName) {
```

Comment:
interface should be general. like collectionName, although could get from entityClass, but still use have choice to customize it